OHM-Trade-Agent-v1/app/services/qualified_alert_outbox.py

```
@@ -0,0 +1,354 @@
+        ).upper()
+        if direction not in {"LONG", "SHORT"}:
+            raise ValueError("qualified alert outbox direction is invalid")
+        action = str(row.get("action") or "")
+        candidate = dict(row.get("tracking_candidate") or {})
+        leverage = float(
+            row.get("leverage")
+            or (2.0 if direction == "SHORT" else 1.0)
+        )
+        if not math.isfinite(leverage) or leverage <= 0:
+            raise ValueError("qualified alert outbox leverage is invalid")
+    except Exception:
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)